### PR TITLE
feat(TradeChart): wrapped library

### PR DESCRIPTION
**Pull request by zachdt**
Using `lightweight-charts-wrapper` for a variety of reasons, mostly QoL and stability.

includes various formatting improvements

### CODE REVIEW

**Comment by zachdt** [comment]
Will merge when I am able to fix pricing in `rmm-indexer` to finalize timescale and pricescale margins.

**Comment by zachdt** [comment]
https://github.com/tradingview/lightweight-charts/issues/1598

Looks like `lightweight-charts-wrapper` wont see this update unless I get a PR merged in.  Holding off until this patch is merged.

**Review by Alexangelj — APPROVED** [review]

**Pull request merged**
